app/containers/ViewQuestion/QuestionTitle.jsx

```diff
@@ -48,6 +49,17 @@ const QuestionName = H3.extend`
 const Div = styled.div`
   min-width: 140px;
 `;
+const TitleContainer = styled.div`
+  display: flex;
+  align-items: center;
+  .bountyActive {
+  }
+  .bountyPaid {
+    background-color: green;
```

Comment:
Use constants for colors. You can change community config and look at how your changes are in another community style